packages/appeals-service-api/src/services/confirm-email.service.js

```diff
@@ -0,0 +1,38 @@
+const logger = require('../lib/logger');
+const mongodb = require('../db/db');
+const { sendConfirmEmailAddressEmail, createToken } = require('../lib/notify');
+
+const confirmEmailCreateService = async (appeal) => {
+  const token = createToken();
+  const query = { appealId: appeal.id };
+  const option = { upsert: true };
+  try {
+    await mongodb
+      .get()
+      .collection('confirmEmail')
+      .updateOne(
+        query,
+        {
+          $set: {
+            token,
+            tokenStatus: 'NOT_SENT',
+            appealId: appeal.id,
```

Comment:
I don't think this is necessary as the appeal retrieval is done later. We want to store the token when sending the e-mail just to confirm it's legit when coming back